src/mlpack/methods/nca/nca_main.cpp

```diff
@@ -224,7 +224,7 @@ void BINDING_FUNCTION(util::Params& params, util::Timers& timers)
   if (normalize)
   {
     // Find the minimum and maximum values for each dimension.
-    arma::vec ranges = arma::max(data, 1) - arma::min(data, 1);
+    arma::vec ranges = max(data, 1) - arma::min(data, 1);
```

Comment:
```suggestion
    arma::vec ranges = range(data, 1);
```
I think you can actually use `arma::range()` here instead, it's a little cleaner (and I believe it will be incrementally faster, but that's not so big a deal at this part of the code).